### ISSUE TITLE
Docker Login Isuue

### ISSUE THREAD

**Issue by sunny1610** [docker/hub-feedback#1642]
I have installed docker and trying to login to docker through command line.I am not able to login.
Below is the error I am facing
C:\>docker login
Authenticating with existing credentials...
Login did not succeed, error: Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.
Username (sunny1610): sunny1610
Password:
Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)

I have kept my docker-compose file in C drive
Please suggest

**Comment by jmwong** [comment]
@sunny1610 sorry for your inconvenience. are you able to make a request to  https://registry-1.docker.io/v2/ using other mechanisms or from a different network?

**Comment by sunny1610** [comment]
No I am not able to request from other network too.
When I type docker login command and enters Username and Password it is giving the below error

Login did not succeed, error: Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.

What should I do in this case.Is there any configuration missing or any specific path where I should keep my Config file.Please suggest

**Comment by jmwong** [comment]
Unfortunately this looks like a connectivity issue on your client as we’ve not observed similar issues on our end. Would suggest trying a VPN.

**Comment by EdDeAlmeidaJr** [comment]
I am getting the same here:

> Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)

No proxy. No VM. Just my (Debian) Linux box connected directly to the web.

**Comment by mannydr3** [comment]
Same issue here:
Client: Docker Engine - Community
 Version:           18.09.0
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        4d60db4
 Built:             Wed Nov  7 00:47:51 2018
 OS/Arch:           windows/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.0
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.4
  Git commit:       4d60db4
  Built:            Wed Nov  7 00:52:55 2018
  OS/Arch:          linux/amd64
  Experimental:     false

1. I'm signed on with Docker for windows using my user name (found out that using your email address works to get in on the hub etc... but won't work when using it with PowerShell.  I signed out both from PS and the app on my machine. Then, signed in with my username/password, not email address/password.
That did work up until recent. Now I'm getting this when it used to work:

Authenticating with existing credentials...
docker : Login did not succeed, error: Error response from daemon: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 
192.168.137.1:53: read udp 192.168.137.31:50273->192.168.137.1:53: i/o timeout.

2. I have not made any changes to any v-switches or adapters. Everything was as-is and working up until I proceeded to lesson 4 in the Documentation (https://docs.docker.com/v17.09/get-started/part4/#deploy-the-app-on-the-swarm-manager), after creating the swarm and Deploy the app on the swarm manager is what I'm stuck on since it's not taking my credentials, even after manually doing so. Help anyone?
